lib/src/isolate_channel.dart

```diff
@@ -49,7 +49,7 @@ class IsolateChannel<T> extends StreamChannelMixin<T> {
     // The first message across the ReceivePort should be a SendPort pointing to
     // the remote end. If it's not, we'll make the stream emit an error
     // complaining.
-    var subscription;
+    StreamSubscription<T> subscription;
     subscription = receivePort.listen((message) {
```

Comment:
The declaration of `subscription` is separate from it's initialization, so its later use (on line 57) needs to be guarded against null.